mobile/src/main/java/rs/readahead/washington/mobile/views/fragment/MicFragment.kt

```diff
@@ -370,6 +370,7 @@ class MicFragment : MetadataBaseLockFragment(),
         mRecord.background =
             AppCompatResources.getDrawable(requireContext(), R.drawable.red_circle_background)
         mRecord.setImageResource(R.drawable.stop_white)
```

Comment:
```
mRecord.apply {
background =
            AppCompatResources.getDrawable(requireContext(), R.drawable.red_circle_background)
setImageResource(R.drawable.stop_white)
contentDescription = getString(R.string.action_stop)
}
```

Using the apply function in this way allows you to group the property assignments together, making the code more concise and readable. It eliminates the need to repeatedly reference mRecord for each property assignment.

**Please apply the feedback on other parts of the code if it's applicable!**